DscResources/MSFT_xMySqlUser/MSFT_xMySqlUser.schema.mof

```diff
@@ -2,11 +2,11 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlUser")] 
 class MSFT_xMySqlUser : OMI_BaseResource
 { 
-  [Key, Description("Name of MySQL user to create or remove.")] String Name;
-  [Required, Description("Credential for MySql user."), EmbeddedInstance("MSFT_Credential")] String Credential;
-  [Required, Description("MySql connection credential used to create a user."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
-  [Write, Description("Ensure mysql user is present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-
+    [Key, Description("Name of MySQL user to create or remove.")] String UserName;
+    [Required, Description("Credential for MySQL user."), EmbeddedInstance("MSFT_Credential")] String UserCredential;
+    [Required, Description("MySQL root credential used to create a user."), EmbeddedInstance("MSFT_Credential")] String RootCredential;
+    [Write, Description("Ensure MySQL user is present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Required, Description("MySQL Version Number")] String MySqlVersion;
```

Comment:
If you have several setups though with different resource setups and different MySQL versions you would need it to verify that you have the correct versions installed with each application group, also on the flipside if you update the resource you would not want your version of MySQL to accidentally change since you are updating your resources to keep up with potential bugs and security issues.  
so as an example my resource specifies a default of MySQL 5.6.13 which is fine because when I made my config files my app requires 5.6.10 - 5.6.13. Later on the resource updates for 5.7 and changes the default to 5.7, I update my resource files and now my SQL server goes to 5.7 and my app happens to break on MySQL 5.7. If I started by making the resource say it explicitly needed 5.6.13 I would not get bit by that. So why add a bug that has no real reason?